pkgs/by-name/go/goat-cli/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib,
+fetchFromGitHub,
+buildGoModule,
+fetchpatch
+}:
+
+buildGoModule rec {
+  pname = "goat-cli";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    repo = "goat";
+    owner = "studio-b12";
+    rev = "v${version}";
+    hash = "sha256-0rOGZEMSPIU3gK31X8uozhMh/82HGbcCpGOGKZ0qvrc=";
+  };
+
+  vendorHash = "sha256-DtEXgGYSkWO876so6LEOkhVwDt/zrflDZdY4O2lz1mw=";
+
+  patches = [
+    ./mock-fix.patch
+  ];
```

Comment:
Maybe I'm missing something, but the patch doesn't seem to do anything because `doCheck = false;` is set? I removed both the patch and these lines, and it builds and runs. Else it LGTM.

Reply:
Without the patch you should be running into an issue with an undefined import during the `checkPhase`. I've opt'd to just disable checks for the time being in https://github.com/NixOS/nixpkgs/pull/275801/commits/d8f88ee8bd2dad109acd79085b0c75b209917830